plugins/native/CMakeLists.txt

```diff
@@ -119,7 +119,7 @@ f3d_plugin_declare_reader(
   EXTENSIONS vtp vtu vtr vti vts vtm
   MIMETYPES application/vnd.vtp application/vnd.vtu application/vnd.vtr application/vnd.vti application/vnd.vts application/vnd.vtm
```

Comment:
should this be split ?

Reply:
I have split them, but I use `vtkXMLGenericDataObjectReader` in each reader. Do you think it's fine or should I specialize it? (e.g. using `vtkXMLPolyDataReader` for vtp files)

Reply:
The goal of splitting them is to have a different format description